backend/src/server/routes/v1/project-router.ts

```diff
@@ -236,6 +236,45 @@ export const registerProjectRouter = async (server: FastifyZodProvider) => {
     }
   });
 
+  server.route({
+    url: "/:workspaceId/overview",
+    method: "POST",
+    config: {
+      rateLimit: writeLimit
+    },
+    schema: {
+      params: z.object({
+        workspaceId: z.string().trim()
+      }),
+      body: z.object({
+        name: z.string().trim(),
+        description: z.string().trim()
+      }),
+      response: {
+        200: z.object({
+          message: z.string(),
+          workspace: SanitizedProjectSchema
+        })
+      }
+    },
+    onRequest: verifyAuth([AuthMode.JWT]),
+    handler: async (req) => {
+      const workspace = await server.services.project.updateOverview({
+        actorId: req.permission.id,
+        actor: req.permission.type,
+        actorAuthMethod: req.permission.authMethod,
+        actorOrgId: req.permission.orgId,
+        projectId: req.params.workspaceId,
+        name: req.body.name,
+        description: req.body.description
+      });
+      return {
+        message: "Successfully changed workspace overview",
+        workspace
+      };
+    }
+  });
+
```

Comment:
curious; what was the reason for adding this endpoint? Can't we just use patch workspace/project?

Reply:
there were previous endpoints dedicated to patching just the name
the patch project seems to be more for Public API than internal, and thus didnt want to re-use in UI when separate already existed